R/utils_render_common.R

```diff
@@ -288,7 +288,9 @@ perform_col_merge <- function(data,
         )
     }
 
-    if (type == "merge_range") {
+    if (type %in% c("merge_range", "merge_uncert")) {
```

Comment:
Rather than a conditional here, validate that `type` is a valid value at the beginning of the function

Reply:
Now set to throw if `type` is invalid. An additional safeguard for checking `type` is now in the `dt_col_merge_entry()` function.